src/Symfony/Component/Console/Output/StreamOutput.php

```diff
@@ -70,14 +69,12 @@ public function getStream()
      */
     protected function doWrite($message, $newline)
     {
+        $message = (string) $message;
```

Comment:
It's not related to the fix, I'm going to revert this change while merging the PR.